tests/stringparsingcheck.cpp

```diff
@@ -12,7 +12,7 @@
 #define JSON_TEST_STRINGS
 #endif
 
-#ifndef _MSC_VER
+#if (!(_MSC_VER) && !(__MINGW32__) && !(__MINGW64__))
```

Comment:
Given how many places we do this,is there a macro that makes sense and covers these three cases? Better yet, perhaps we should have a <simdjson_dirent.h> somewhere, and get rid of the if entirely :)

Eh. For now it's fine, we should really do the simdjson_dirent.h thing regardless of this patch (which suggests it should be done outside of this patch). I'll file an issue.